docs/root/configuration/http/http_filters/ip_tagging_filter.rst

```diff
@@ -3,9 +3,9 @@
 IP Tagging
 ==========
 
-The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* with the string tags for the trusted address from
-:ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`. If there are no tags for an address,
-the header is not set.
+The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* if the alternate :ref: `ip_tag_header <envoy_v3_api_msg_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
```

Comment:
I'd rephrase 
sets the header *x-envoy-ip-tags* or the provided `ip_tag_header ...` with the string tags